cmd/sealos/cmd/exec.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2022 cuisongliu@qq.com.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/labring/sealos/pkg/ssh"
```

Comment:
github.com/labring/sealos/pkg/ssh   => github.com/labring/sealos/pkg/exec , github.com/labring/sealos/pkg/scp

Reply:
Since `exec.go` depends on `pkg/clusterfile`, a cyclic import occurred. Not sure how to resolve this?